src/features/user/ManagePayouts/screens/EditBankAccount.tsx

```diff
@@ -0,0 +1,156 @@
+import { ReactElement, useContext, useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import { styled } from '@mui/material/styles';
+import i18next from '../../../../../i18n';
+import Link from 'next/link';
+import { putAuthenticatedRequest } from '../../../../utils/apiRequests/api';
+import { usePayouts } from '../../../common/Layout/PayoutsContext';
+import { ErrorHandlingContext } from '../../../common/Layout/ErrorHandlingContext';
+import BankDetailsForm, { FormData } from '../components/BankDetailsForm';
+import BackArrow from '../../../../../public/assets/images/icons/headerIcons/BackArrow';
+import CustomSnackbar from '../../../common/CustomSnackbar';
+import CenteredContainer from '../../../common/Layout/CenteredContainer';
+import { UserPropsContext } from '../../../common/Layout/UserPropsContext';
+import isApiCustomError from '../../../../utils/apiRequests/isApiCustomError';
+import { PayoutCurrency } from '../../../../utils/constants/payoutConstants';
+
+const { useTranslation } = i18next;
+
+const FormHeader = styled('div')(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  gap: 16,
+  marginBottom: 24,
+  width: '100%',
+  '& .formTitle': {
+    color: theme.palette.text.secondary,
+    fontSize: theme.typography.h2.fontSize,
+  },
+}));
```

Comment:
create a separate file for style

Reply:
I'm going to be going through a few features in the dashboard and refactor places where reuse is possible, or code can be split for simplicity. That will be part of a separate PR.